src/main/java/sirius/biz/jobs/batch/file/ArchiveExportJob.java

```diff
@@ -79,5 +83,28 @@ protected void collectParameters(Consumer<Parameter<?, ?>> parameterCollector) {
             super.collectParameters(parameterCollector);
         }
     }
+
+    /**
+     * Digests every entry of a fresh created export archive.
+     * <p>
+     * Override this method in order to perform validations on contents of the final archive.
```

Comment:
should this be overwritten? this would remove the logic here?

Reply:
shouldn't we separate validaten logic from store logic? it its no clear when what has to be invoked..